packages/aeproject-cli/aeproject-contracts/aeproject-contracts.js

```diff
@@ -30,27 +48,53 @@ const mainFlow = async (options) => {
     });
 };
 
-const checkGlobalModuleExistence = (stdout, moduleName) => {
+const checkGlobalModuleExistence = async (stdout, moduleName) => {
     const devDependenciesPath = stdout.split('\n')[0];
     nodeModulesPath = path.join(devDependenciesPath, contractsConstants.NODE_MODULES_LITERAL);
     const modulesContent = fs.readdirSync(nodeModulesPath);
-    return modulesContent.includes(moduleName);
+
+    let isContractsInstalled = false;
+    // Travis throw exception when there are NO installed packages
+    let tempCwd = process.cwd();
+    try {
+        let yarnNodeModulePath = await getYarnNodeModulePath();
+        process.chdir(yarnNodeModulePath);
+
+        let yarnLS = await exec('ls');
+        isContractsInstalled = yarnLS.stdout.split('\n').indexOf('contracts-aepp') >= 0;
+    } catch (error) {
+        // When there is no installed package/directory is missing/, tests on travis throw error
+    }
+
+    process.chdir(tempCwd);
+
+    return modulesContent.includes(moduleName) || isContractsInstalled;
 };
 
 const installRepo = async () => {
+    const currentPath = process.cwd();
     console.log('====== Installing Contracts web Aepp ======');
-    await exec(`npm install -g ${ contractsConstants.CONTRACTS_AEPP_GITHUB_URL }`);
+
+    await exec(`yarn global add ${ contractsConstants.CONTRACTS_AEPP_GITHUB_URL }`);
```

Comment:
why not yarn install?

Reply:
and in the previous version, we install 'contracts' globally